### PR TITLE
Dto integration

### PR DESCRIPTION
**Pull request by mykolaKhlopyk**

### CODE REVIEW

File: rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/controller/library/CreateNewLibraryNatsController.kt [hunk 1]
```diff
@@ -27,13 +27,13 @@ class CreateNewLibraryNatsController(private val libraryService: LibraryService)
             createFailureResponse(it)
         }
 
-    private fun createdLibrary(request: CreateNewLibraryRequest): LibraryOuterClass.Library {
+    private fun createdLibrary(request: CreateNewLibraryRequest): Library {
         val createdLibrary =
             libraryService.createNewLibrary(request.libraryName, request.telegramUserId).toLibraryResponse()
         return createdLibrary
     }
```
Comment (SlavikN14) [inline rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/controller/library/CreateNewLibraryNatsController.kt]: If you use a kotlin-like code style in your class, it is better to stick to it in all functions

```kotlin
 private fun createdLibrary(request: CreateNewLibraryRequest): Library =
            libraryService.createNewLibrary(request.libraryName, request.telegramUserId).toLibraryResponse()
```

File: rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/mapper/MapperEntityToResponse.kt [hunk 2]
```diff
@@ -1,29 +1,37 @@
...
         .setAdditionInfoAboutWord(
-            additionalInfoAboutWord.let {
-                WordOuterClass.AdditionalInfoAboutWord.newBuilder()
+            additionalInfoAboutWordDtoResponse.let {
+                AdditionalInfoAboutWord.newBuilder()
                     .setDefinitionOfWord(it.definitionOfWord)
                     .setPronunciationOfWord(it.pronunciationOfWord)
                     .setExampleInSentences(it.exampleInSentences)
                     .setLinkToAudio(it.linkToAudio)
             })
```
Comment (SlavikN14) [inline rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/nats/mapper/MapperEntityToResponse.kt]: Wouldn't it be better to put this in a separate function?

File: rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/service/WordService.kt [hunk 3]
```diff
@@ -25,37 +35,52 @@ class WordServiceImpl(
...
+            .toDtoResponse()
     }
 
-    override fun deleteWord(libraryName: String, telegramUserId: String, wordSpelling: String): Word {
+    override fun deleteWord(libraryName: String, telegramUserId: String, wordSpelling: String): WordDtoResponse {
         val wordId =
             wordRepository.getWordIdByLibraryNameTelegramUserIdWordSpelling(libraryName, telegramUserId, wordSpelling)
         return wordRepository.deleteWord(wordId)
+            .toDtoResponse()
     }
```
Comment (SlavikN14) [inline rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/service/WordService.kt:77]: Isn't looks better?

```kotlin
override fun deleteWord(libraryName: String, telegramUserId: String, wordSpelling: String): WordDtoResponse =
        wordRepository.run {
            deleteWord(getWordByLibraryNameTelegramUserIdWordSpelling(libraryName, telegramUserId, wordSpelling).id)
                .toDtoResponse()
        }
 }
```

Comment (mykolaKhlopyk) [reply rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/service/WordService.kt:77]: it looks better, yeap, but becomes harder to read

**Review by SlavikN14 — APPROVED** [review]

File: rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/service/WordService.kt [hunk 4]
```diff
@@ -25,37 +35,52 @@ class WordServiceImpl(
     val additionalInfoAboutWordService: AdditionalInfoAboutWordService
 ) : WordService {
 
-    override suspend fun saveNewWord(libraryName: String, telegramUserId: String, wordDto: WordDto): Word {
+    override suspend fun saveNewWord(
```
Comment (ajax-gnatiuk-s) [inline rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/service/WordService.kt:38]: `suspend`? Do u use coroutines?

**Review by ajax-gnatiuk-s — APPROVED** [review]

File: rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/service/WordService.kt (hunk 4 above)
Comment (mykolaKhlopyk) [reply rest/src/main/kotlin/systems/ajax/englishstudytelegrambot/service/WordService.kt:38]: i rewrite to reactor now, and delete this `suspend`

**Pull request merged**